aiida/orm/nodes/data/list.py

```diff
@@ -77,15 +77,18 @@ def remove(self, value):
         del self[value]
 
     def pop(self, **kwargs):  # pylint: disable=arguments-differ
+        """Remove and return item at index (default last)."""
```

Comment:
That function does not seem to return anything or am I wrong?

Reply:
Hmm, yeh good point it should actually return; thats a bug in the code I would say.

Its a bit out of scope of this PR, so perhaps we should fix in another small PR.
(I only actually made this change to fix a warning by sphinx about the inherited docstring)

Reply:
Up to you - in any case, the docstring should be consistent and if you prefer not to fix it here, we should open an issue

Reply:
ok I've fixed it here then 446d1a3